Framework/Algorithms/src/FilterEvents.cpp

```diff
@@ -1143,17 +723,18 @@ void FilterEvents::createOutputWorkspacesSplitters() {
       }
       optws->setComment(info);
       optws->setTitle(info);
-    } // END-IF infor WS
+    }
 
-    // Add to output properties.  There shouldn't be any workspace
-    // (non-unfiltered) skipped from group index
+    //
+    // Declare the filtered workspace as an output property.
+    // There shouldn't be any non-unfiltered workspace skipped from group index
     if (add2output) {
-      // Generate output property name
-      std::stringstream propertynamess;
-      if (wsgroup == -1) {
-        propertynamess << "OutputWorkspace_unfiltered";
+      // Generate the name of the output property
```

Comment:
```suggestion
      // Generate the name of the output property. Only workspaces that are set as output properties get history added to them
```

Reply:
What do you mean with "history". Is this in the picture?
![Capture](https://user-images.githubusercontent.com/1136006/230919139-0434c6d4-803f-4e51-a121-831ff40c8e24.PNG)

Reply:
Correct. Workspaces that aren't labeled through output parameters in an algorithm, don't get workspace history set. This is a clarification on the comment so developers understand why it is being done